mentoring/mentoring.py

```diff
@@ -444,7 +524,7 @@ def studio_submit(self, submissions, suffix=''):
                 'result': 'error',
                 'message': e.message
             }
-            success = False
+
```

Comment:
@Kelketek nit: probably `success=True` a [couple of lines above](https://github.com/edx-solutions/xblock-mentoring/pull/104/files#diff-957657c893cbf519c26a2f59268a437dR518) should be moved into `else` block.